README.md

```diff
@@ -10,12 +10,51 @@ Send a HTML or Mustache template and recieve a PDF stream as the response.
 
 ## Install and start
 
-### Node App
+### Docker container
```

Comment:
Maybe something along the lines of 'Node App - Running a local html-pdf-instance in a docker container'?